examples/aws-lambda/simple-app/async/handled-exception/async-handled-exception.js

```diff
@@ -0,0 +1,25 @@
+const Bugsnag = require('@bugsnag/js')
+const BugsnagPluginAwsLambda = require('@bugsnag/plugin-aws-lambda')
+
```

Comment:
Just one request. Can you add an example in which we call multiple functions and in those function error could occur but they should not break the flow of execution? 

This in regard use case when failure in some place is OK but still be need log regarding those failures - One API request